redisbp/hooks.go

```diff
@@ -4,10 +4,11 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/go-redis/redis/v7"
-
 	"github.com/reddit/baseplate.go/batcherror"
 	"github.com/reddit/baseplate.go/tracing"
+
+	"github.com/go-redis/redis/v7"
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
Shouldn’t these be before the baseplate imports? Or am I mixing that up with the Python conventions ?

Reply:
I think them as this way: stdlibs (1st party), same module (2nd party), and 3rd parties. that's why I _slightly_ prefer this order. but as long as the latter 2 are separated (except in examples, in examples we are in user's shoes so everything is 3rd party to them) I'm ok with it. if python programmers find it annoying I can switch it back :sweat_smile:

Reply:
I think the other way is probably preferable? goimports even has a flag to support it (maybe we should use it?)

https://github.com/golang/tools/blob/e1da425f72fd3793b579f4e74d908ba96eb16c8a/cmd/goimports/goimports.go#L61

Reply:
that option is likely from pre-modules days (as it accepts a list of prefixes), but fine, I'll switch back.

Reply:
Done.